web/src/components/FileBrowser/FileUploadInstructions.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-sheet
+    class="d-flex"
+    height="70vh"
+  >
+    <v-row
+      class="d-flex flex-column justify-center text-center"
+    >
+      <div class="text-h6 font-weight-light">
+        This Dandiset does not currently have any files associated with it,
+        but this is where they will appear once they're added.
+      </div>
+      <div class="my-7">
+        <span class="text-subtitle-1">Use the DANDI CLI on the command line:</span>
+        <div
+          class="d-flex justify-center"
+          style="font-family: monospace;"
+        >
+          <v-sheet
+            color="black"
+            width="60%"
+            class="white--text pl-2 py-1 text-left"
+          >
+            <div>> dandi download https://dandiarchive.org/dandiset/{{ dandisetIdentifier }}/draft</div>
+            <div>> cd {{ dandisetIdentifier }}</div>
+            <div>> dandi organize &lt;source_folder&gt; -f dry</div>
+            <div>> dandi organize &lt;source_folder&gt;</div>
+            <div>> dandi upload</div>
+          </v-sheet>
+        </div>
+      </div>
+      <div>
+        <span class="text-subtitle-1">Don't have DANDI CLI?</span>
+        <div>
+          <span class="text-body-2 grey--text text--darken-1">
+            Install the Python client (DANDI CLI) in a Python 3.7+ environment using command:
+          </span>
+        </div>
+        <div
+          class="d-flex justify-center"
+          style="font-family: monospace;"
+        >
+          <v-sheet
+            color="black"
+            width="60%"
+            class="white--text pl-2 py-1 text-left"
+          >
+            <span>> pip install "dandi>=0.13.0"</span>
+          </v-sheet>
+        </div>
+      </div>
+    </v-row>
+  </v-sheet>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useDandisetStore } from '@/stores/dandiset';
+
+const store = useDandisetStore();
+const dandisetIdentifier = computed(() => store.dandiset?.dandiset.identifier);
```

Comment:
There's a small typing problem here. `dandisetIdentifier` has type `string | undefined` but you're assuming it's defined at the point of use in the template.

Reply:
Fixed https://github.com/dandi/dandi-archive/pull/1342/commits/5c98f19e2097faa605b16ebf80de5a3a21c86989